docker/Dockerfile

```diff
@@ -1,19 +1,38 @@
-FROM python:2.7
+FROM python:2.7.15
```

Comment:
Any reason why to lock into a minor version?

Reply:
It's unlikely, but something could shift slightly if we don't pin it (for example, python 2.7.9 - ish changed the way that ssl worked a bit). Also it ensures we are using the exact version of python that we are using in Heroku. 

However, it's unlikely to be an issue, and I have no problem removing the minor version pin if others feel strongly about it.